src/savegame_json.cpp

```diff
@@ -452,6 +452,8 @@ void Character::load( const JsonObject &data )
     data.read( "base_height", init_height );
     if( !data.read( "blood_type", my_blood_type ) ) {
         randomize_blood();
+    } else {
+        data.read( "blood_rh_factor", blood_rh_factor );
     };
```

Comment:
```suggestion
    if( !data.read( "blood_type", my_blood_type ) ||
        !data.read( "blood_rh_factor", blood_rh_factor ) ) {
        randomize_blood();
    };
```
After the last update, `blood_type` is in the save file, but `blood_rh_factor` is not, it would be nice to check both of them.